applications/LiisaPortfolioTracker.md

```diff
@@ -0,0 +1,269 @@
+
+#### 2) Data Enrichment
+
+Our backend serves as a vital nexus between supported blockchain protocols and valuable metrics we generate for user wallets and NFT collections. It involves three key activities: crafting and executing API queries, initiating timed triggers, and developing computational algorithms for metrics computation. We create and execute precise API queries to extract event data from Subscan for protocols such as Astar, Basilisk, Efinity, Moonbeam, and Unique Network. This enables the continuous refresh of our system with recent blockchain events pertinent to the monitored NFT collections and user wallets. 
+
+Our backend employs timed triggers to initiate queries to Subscan at predefined intervals, ensuring we maintain an up-to-date snapshot of relevant blockchain events. Incoming event data, acquired via timed or event-driven triggers, is processed using specially designed computational algorithms. These algorithms generate key performance indicators (KPIs) and metrics for both user wallets and NFT collections, capturing the most essential insights. The calculated metrics are persistently stored in our database, ensuring their continuous availability for subsequent retrieval, analysis, and the provision of comprehensive insights into users' wallets and NFT collections.
+
+All the code for data enrichment will be open-source so that more developers in the ecosystem can leverage our processes for retrieving NFT data from subscan API and wallets, as well as creating NFT metrics.
+
+#### 3) Database 
+
+We will be utilizing PostgreSQL as our database management system to store and manage all data. PostgreSQL is an advanced open-source relational database management system that offers a broad range of powerful features. As our choice of infrastructure provider, we will be deploying PostgreSQL on Amazon Web Services (AWS).
+
+#### 4) Frontend
+
+Within the context of the MVP, we'll utilize bubble.io for frontend development to accelerate development speed and iterate quickly. This approach will help us create a functional product that meets user needs faster. Upon achieving a working product, at a later stage we will transition to react.js for enhanced flexibility and performance.
+
+#### 5) Bubble.io plug-ins
+
+We will develop open-source plug-ins for Bubble.io, enabling connections to the selected wallets: Subwallet, Parity signer, Polkadot-js, Subsquid, Fearless wallet, Nova Wallet, Talisman. Built using Node.js, these plug-ins will allow for the users to log-in to the platform using their wallets, and will facilitate the retrieval of information about the NFTs owned by the user to compute the portfolio metrics. By open-sourcing these plug-ins, we intend to improve access from the growing no-code community to the Polkadot ecosystem.
```

Comment:
How would your polkadot js plugin differ from the existing: https://bubble.io/plugin/polkadot-wallet-1639402639641x977692461648052200